README.md

```diff
@@ -1,33 +1,68 @@
 <div align="center">
-<img src="./docs/assets/logo.png" alt="Margarita" title="Margarita" width="300"/>
+  <a href="http://margarita.kiwi.com"><img src="./docs/assets/logo.png" alt="Margarita" title="Margarita" width="200" /></a>
 </div>
+<h1 align="center">Margarita</h1>
+<h4 align="center">Multi-platform <span title="travel">✈️</span> travel app  for iOS, Android and Web<br></h4>
 
-# Margarita
+ <p align="center">built with <strong>React Native, GraphQL and <a href="https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/" target="_blank">Tequila API</a></strong></p>
 
-> Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
+## Demo
 
-This project is a living example of a client app you could start building with Tequila.
+Live demo: https://kiwicom-margarita.netlify.com (web version)
```

Comment:
We can also consider adding Expo link for mobile version https://expo.io/@kiwicom-margarita-public/tequila-client

Reply:
Good idea. 
We should change this: 
<img width="307" alt="Screenshot 2019-05-29 at 10 08 23" src="https://user-images.githubusercontent.com/858321/58540513-d14f8480-81f9-11e9-98e9-34ca566e7cf2.png">

Reply:
Done

Reply:
Definitely. I've already created separate issue for it https://github.com/kiwicom/margarita/issues/849